src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: #{$transition};
```

Comment:
specify for which props you’re applying transition

Reply:
added